lib/app_manifest/serializer.rb

```diff
@@ -0,0 +1,34 @@
+module AppManifest
+  # Serialization logic for AppManifest models
+  module Serializer
```

Comment:
This is fixed. Instead of automatically coercing unspecified keys to something, we now allow attributes like `environments` and `addons` to be `nil`. 

This isn't perfect, as something like `manifest.addons.map(&:plan)` would now fail.